README.md

```diff
@@ -95,6 +95,6 @@ repeat these steps to upgrade to future releases.
 ## Building
 
 * [Linux](docs/build-instructions/linux.md)
-* [OS X](docs/build-instructions/os-x.md)
+* [macOS](docs/build-instructions/macos.md)
```

Comment:
IMHO, the day on which we'll have to change it will definitely come. And the longer we wait, the more links will break...

@thomasjo?